macos/FluentUI/Button/Button.swift

```diff
@@ -424,7 +424,12 @@ class ButtonCell: NSButtonCell {
 			y += CGFloat(yOffsetSign) * (titleSize.height + titleToImageSpacing - titleToImageVerticalSpacingAdjustment) / 2
 		}
 
-		return NSRect(x: x, y: y, width: imageSize.width, height: imageSize.height)
+		return NSRect(
+			x: x.rounded(.towardZero),
+			y: y.rounded(.towardZero),
+			width: imageSize.width.rounded(.toNearestOrAwayFromZero),
+			height: imageSize.height.rounded(.toNearestOrAwayFromZero)
```

Comment:
Don't we always just want the width/height dimensions rounded away from zero to err on the side of extra space over compression/truncation?